api/class2.js

```diff
@@ -83,13 +259,51 @@ router.put('/customers/:id', function(req, res) {
 //   room_price: 129.90
 // }
 // TODO: add code here
+// get '/reservations/active-on/:date'
+// TODO: add code here
+router.get('/reservations/check_in_date/:check_out_date', function (req, res) {
+
+  var sql = `select * from reservations where check_out_date=?`;
+
+  db.all(sql, [req.params.check_out_date], (err, rows) => {
 
+    if (err) {
+      console.log('ERROR fetching from the database:', err);
+      return;
+    }
+
+    console.log('Request succeeded, new data fetched', rows);
+    res.status(200).json({
+
+      reservations: rows
+    });
+  });
+
+});
 
 // get `/detailed-invoices'
 // TODO: add code here
 
 
-// get `/reservations/details-between/:from_day/:to_day`
+router.get(`/reservations/active-on/:chech_in_date`, function (req, res) {
```

Comment:
chech_in_date should be check_in_date

Reply:
actually, that's misleading - it should just be called active_date or something.